test-cases/longevity/longevity-alternator-streams-100gb-3h.yaml

```diff
@@ -0,0 +1,55 @@
+test_duration: 200
+
+prepare_write_cmd:
+  - >-
+    hydra-kcl -t usertable_streams --timeout 5000
```

Comment:
check results to see if 5000 is enough time for KCL to stream all data to target table.